config-devel.toml

```diff
@@ -1,6 +1,7 @@
 [broker]
 address = "kafka:29092"
 topic = "ccx.ocp.results"
+publish_topic = "platform.payload-status"
```

Comment:
I know that this is a WIP, so sorry if it is a nitpick.

In theory, the Kafka server can be different than the consumer one, so maybe it deserves a new section in the configuration file for the payload tracker integration where the address and topic are configured

Reply:
That makes sense, I guess. It is getting a bit crowded in the `broker` section. I'd rather add these tweaks in a separate PR though because this one already requires a lot of changes to many different files, so the longer I keep it open, the more conflicts I have to resolve every day.